src/states/DropdownState.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+    <div>
+        <slot :context="this"></slot>
+    </div>
+</template>
+
+<script>
+module.exports = {
+    name: `DropdownState`,
+    props: {
+        value:{
+            type: [Number, String, Object, Array]
+        },
+        disable: {
+            type: Boolean,
+            default: () => false
+        },
+        options: {
+            type: Array,
+            default: () => []
+        },
+        multiply: {
+            type: Boolean,
+            default: () => false
+        },
+        autoClose: {
+            type: Boolean,
+            default: () => true
+        },
+        validators: {
+            type: Object,
+            default: () => {}
+        },
+        validateHost: {
+            type: Object
+        }
+    },
+    data () {
+        return {
+            open: false,
+            isValid: false,
+            selectedOptions: null
```

Comment:
May be new Set() by default. You don't need create it after it.

Reply:
done